web/blog/2024-10-23-pilcrow-os-maintainers-interview.md

```diff
@@ -0,0 +1,92 @@
+---
+title: 'The Faces Behind Open Source Projects: Pilcrow'
+authors: [milica]
+image: /img/os-interviews/pilcrow.jpeg
+tags: [webdev, auth, os-maintainers]
+---
+
+import Link from '@docusaurus/Link';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+import InBlogCta from './components/InBlogCta';
+import WaspIntro from './_wasp-intro.md';
+import ImgWithCaption from './components/ImgWithCaption'
+
+We're back with our series of posts in which we interview open-source maintainers and showcase their amazing projects. This week we've talked with [Pilcrow](https://github.com/pilcrowonpaper), an amazing young dev from Japan, famous for his auth library Lucia and love for auth and security. 
+
+[Lucia](https://github.com/lucia-auth/lucia) is one of the open-source tools that we use under-the-hood at Wasp. Unfortunately, this project is going to be deprecated in April 2025. Wasp will make all the necessary changes on time, no worries 😃 You should look into other projects Pilcrow has kickstarted as well, some of those will serve as a replacement for Lucia.
+
+We hope this interview inspires you to contribute to the open-source community yourself. Let's dive in!
+
+- **Could you tell us a little about yourself and how you got into coding? What led you to focus on authentication and tooling for developers?**
+
+I’m Pilcrow. I’m a university student in Japan working on various open source projects around auth and security. I started programming like 4 years ago so I'm still new to the field. I love designing APIs a lot and that's probably the biggest reason I enjoy working on libraries.
+
+- **Lucia brought you recognition within the dev community. What inspired you to create it, and how did you decide to dedicate so much time and effort to it?**
+
+I was asked to build several websites for school events and the part I struggled with the most was auth. Half of it was my inexperience, but I also found third-party offerings like Firebase to be unintuitive. Most of their features and APIs at the time were aimed at SPAs as well and didn't work well with server-rendered applications. The open source offerings weren't great either. To this day, I still don't really know how to actually use NextAuth (now Auth.js) or Passport.js.
+
+Over summer break, I decided to build a more leaner NextAuth for SvelteKit so I could use it for my projects. It was my first open source library but it got some attention in the Svelte community. I made it framework agnostic a little while later and it grew naturally.
+
+Working on it got me hooked on library development, and I found designing APIs and even documenting them to be very enjoyable.
+
+:::tip[Would you star us on GitHub?]
```

Comment:
hmm we usually put CTAs like this on DEV, but not on our blog directly, to keep it clean. We should probably have some CTA in the banner of blog, or just a visible number of stars. Although we could do it, just havent done it so far.